Android.mk

```diff
@@ -275,9 +275,16 @@ HYBRIS_INIT_TARGETS := init
 
 ifeq ($(shell test $(ANDROID_VERSION_MAJOR) -ge 10 && echo true),true)
 # init is split of into early and second stage init starting with android 10
-HYBRIS_INIT_TARGETS := init_second_stage com.android.runtime.release init.rc
+HYBRIS_INIT_TARGETS := init_second_stage init.rc
+ifeq ($(shell test $(ANDROID_VERSION_MAJOR) -eq 10 && echo true),true)
+HYBRIS_INIT_TARGETS += com.android.runtime.release
+else
+HYBRIS_INIT_TARGETS += com.android.runtime
+endif
 endif
 
+HYBRIS_INIT_TARGETS += $(filter ueventd% fstab.%,$(PRODUCT_PACKAGES))
```

Comment:
This looks safe, because `$(PRODUCT_PACKAGES)` on Android 10 doesn't contain any `ueventd*` nor `fstab.*`

I'm not sure about <=Android 9 build envs, and if such packages exist there, will building them not impair anything?

Reply:
Whether there is ueventd* or fstab.* in $(PRODUCT_PACKAGES) is device specific and I have seen such packages listed on some community devices even in earlier Android bases.

Reply:
Ah, Github doesn't email edited comments:) Approving